CameraOverlay/Overlay_Demo.py

```diff
@@ -100,6 +103,11 @@ def on_message(client, userdata, msg):
     elif msg.topic == "power_model/max_speed":
         max_speed = str(msg.payload.decode("utf-8"))
         POWER_MODEL_DATA["max_speed"] = float(max_speed)
+    elif msg.topic == "camera/get_overlays":
```

Comment:
This would mean the overlays can only be changed if an overlay is currently on.

Reply:
yes correct! unless we what overlay which Pi will be using on the server?